### PR TITLE
DOC: remove latex and parallel building

### PR DESCRIPTION
**Pull request by jorisvandenbossche**

### CODE REVIEW

**Comment by jorisvandenbossche** [comment]
This seemed to work on my fork on travis, but not sure why removing the parallel would fix it.

**Comment by jreback** [comment]
yep this looks good. fyi

not exactly sure what is wrong here :>
```
/tmp/doc/source/io.rst:3097: ERROR: Error in "ipython" directive:
invalid option block.
.. ipython:: python
   :suppress:
   import os
   os.remove("data.pkl.compress")
   os.remove("data.pkl.xz")
   os.remove("data.pkl.gz")
   os.remove("s1.pkl.bz2")
/tmp/doc/source/overview.rst:120: WARNING: Include file '/tmp/LICENSE' not found or reading it failed
source/whatsnew/v0.20.0.txt:141: ERROR: Error in "ipython" directive:
invalid option block.
```

**Comment by jorisvandenbossche** [comment]
needs a white line, pushed a fix to this branch

File: doc/source/io.rst [hunk 1]
```diff
@@ -3096,6 +3096,7 @@ The default is to 'infer
 
 .. ipython:: python
    :suppress:
+
```
Comment (jreback) [inline doc/source/io.rst]: same line in v0.20.0.txt as well :>

**Comment by jreback** [comment]
I restarted a couple of times: https://travis-ci.org/pandas-dev/pandas/jobs/209555204

maybe: https://groups.google.com/forum/#!msg/sphinx-dev/MtRf64eGtv4/aOa7FdxkAgAJ

here's a fix (increase stack :>)
```
--- a/docs/users_guide/conf.py
+++ b/docs/users_guide/conf.py
@@ -147,6 +147,8 @@ def parse_flag(env, sig, signode):
 def setup(app):
     from sphinx.util.docfields import Field, TypedField
 
+    increase_python_stack()
+
     # the :ghci-cmd: directive used in ghci.rst
     app.add_object_type('ghci-cmd', 'ghci-cmd',
                         parse_node=parse_ghci_cmd,
@@ -171,3 +173,11 @@ def setup(app):
                             Field('since', label='Introduced in GHC version', names=['since']),
                             Field('static')
                         ])
+
+def increase_python_stack():
+    # Workaround sphinx-build recursion limit overflow:
+    # pickle.dump(doctree, f, pickle.HIGHEST_PROTOCOL)
+    #  RuntimeError: maximum recursion depth exceeded while pickling an object
+    #
+    # Default python allows recursion depth of 1000 calls.
+    sys.setrecursionlimit(10000)
```

**Comment by jreback** [comment]
https://github.com/sphinx-doc/sphinx/pull/2325/files is a better view on this.

**Pull request merged**